### ISSUE TITLE
Jarvis Queries and Results UI Change

### ISSUE THREAD

**Issue by lalitmee** [Harkishen-Singh/Jarvis-personal-assistant#163]
**Queries and Results** Section
![Selection_006](https://user-images.githubusercontent.com/10762218/75755986-c52ebf80-5d55-11ea-9cd2-80d6e5cdf21d.png)

We need to come up with some **mockups** for the same and then we will select the mockup which will be good for the **User Experience** and will implement that.

**Comment by algokun** [comment]
@muskankhedia could you please assign this issue to me. I would like to work with this one. Also brief the ui details like what are the things you guys are expecting.

**Comment by muskankhedia** [comment]
Sure. Go ahead.

**Comment by muskankhedia** [comment]
@algokun How is the implementation going?

**Comment by algokun** [comment]
Yeah. It's going well. I'm redesigning the whole ui. I'll try to submit the ui by today

**Comment by algokun** [comment]
Here's my the mockup:
https://www.figma.com/file/f7EQz5c7NIDcrPjokapkx3/JarvisAssistant?node-id=0%3A1

Here's the live preview:
https://www.figma.com/proto/f7EQz5c7NIDcrPjokapkx3/JarvisAssistant?node-id=2%3A2&scaling=min-zoom

Can you please tell me what to do next and what to include in PR, the screenshots or actual mockup file?

**Comment by muskankhedia** [comment]
@algokun Well Done! You can refer to PR #172 to get an idea on how to create a PR.

**Comment by muskankhedia** [comment]
@algokun Please make the PR and submit it asap, else you won't be getting any points for this.

**Comment by algokun** [comment]
Okay @muskankhedia, i'll submit the pr now.

**Comment by muskankhedia** [comment]
@algokun The designs are well made but can you please change the background colour to white and make the combinations accordingly. I missed this point, by mistake, we have planned to keep the background colour as white.
